### DISCUSSION TITLE
How to remove all except for pinned items without having popus ?!

### DISCUSSION THREAD

**Discussion by sabarish-vm** [hluk/CopyQ#1813] [Q&A] [answered]
The default clear tab command clears the tab but if there are pinned items, it keeps popping up "Unpin items to remove first" is it somehow possible to avoid these popups? 
Or is some kind of custom command possible to achieve this ?

**Comment by hluk** [comment, marked as answer]
Ah, that's a bug. Let me fix that in #1823.

**Reply by sabarish-vm** [reply]
Thanks.
